lib/disability_compensation/providers/intent_to_file/evss_intent_to_file_provider.rb

```diff
@@ -6,11 +6,11 @@
 
 class EvssIntentToFileProvider
   include IntentToFileProvider
-  def initialize(current_user)
-    @service = EVSS::IntentToFile::Service.new(current_user)
+  def initialize(current_user = nil, auth_headers = nil)
+    @service = EVSS::IntentToFile::Service.new(current_user, auth_headers)
   end
 
-  def get_intent_to_file
+  def get_intent_to_file(_type, _client_id, _rsa_key_path)
```

Comment:
Q: Curious what this is for. Does the eVSS side need these args? Noting because at a glance, neither the service nor the spec seems to use these

Reply:
The `EvssIntentToFileProvider` and `LighthouseIntentToFileProvider` share the same method name and number of arguments so they can be called dynamically in the `IntentToFilesController`. On Master it doesn't include the arguments yet, but you can see in the [master branch](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/intent_to_files_controller.rb#L18) where the method is called on the return value of `ApiProviderFactory.intent_to_file_service_provider(@current_user)` when we don't always know which provider is being used.

I agree that it is not ideal to place those unused arguments on the method name, but I thought that was better than some conditional logic in the Controller that uses different method names depending on which provider is used. I think Aurora and Seth took a same approach to the same problem somewhere in RatedDisabilities and I thought it was a decent pattern to use for the 3 to 6 months of limbo while we may need to shift between APIs during the migration.